<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonListen">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>开启/关闭服务</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserChert">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>341</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSend">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>341</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>470</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
